packages/manager/src/manager.ts

```diff
@@ -29,20 +30,20 @@ export class Manager {
   authProvider: AuthProvider
   store: DIDStore
   cache: LinkCache
-  idx: IDX
-  ceramic: CeramicClient
+  dataStore: DIDDataStore
+  ceramic: CeramicApi
   threeIdProviders: Record<string, ThreeIdProvider>
 
   // needs work on wording for "account", did, caip10 etc
   constructor(
     authprovider: AuthProvider,
-    opts: { store?: DIDStore; ceramic?: CeramicClient; cache?: LinkCache }
+    opts: { store?: DIDStore; ceramic?: CeramicApi; cache?: LinkCache, dataStore?: DIDDataStore }
   ) {
     this.authProvider = authprovider
     this.store = opts.store || new DIDStore()
     this.cache = opts.cache || new LinkCache()
-    this.ceramic = opts.ceramic || new CeramicClient(CERAMIC_API)
-    this.idx = new IDX({ ceramic: this.ceramic })
+    this.dataStore = opts.dataStore || new DIDDataStore({ ceramic: new CeramicClient(CERAMIC_API), model: idxModel })
+    this.ceramic = opts.ceramic || this.dataStore.ceramic
```

Comment:
I'm not fully familiar with the expected logic, but assuming we want the DIDDataStore to use the provided `opts.ceramic` instance:

```suggestion
    this.ceramic = opts.ceramic || new CeramicClient(CERAMIC_API)
    this.dataStore = opts.dataStore || new DIDDataStore({ ceramic: this.ceramic, model: idxModel })
```

Reply:
👍